tests/subsys/tracing/prj_uart_ctf.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_ZTEST=y
```

Comment:
I do not see this file being used anywhere

Reply:
yes, I will submit another PR for checking tracing ctf,  so will remove this file firstly.  thanks for your reminding.

Reply:
Have removed prj_uart_ctf.conf , thanks.